tests/system_test.py

```diff
@@ -135,25 +131,25 @@ def test_bot_to_db(
 
         # TODO these expected values are defined in lib/ethpy/ethpy/test_fixtures/deploy_hyperdrive.py
         # Eventually, we want to parameterize these values to pass into deploying hyperdrive
-        expected_timestretch_fp = FixedPoint(scaled_value=_calculateTimeStretch(int(0.05e18)))
+        expected_timestretch_fp = FixedPoint(scaled_value=_calculateTimeStretch(FixedPoint("0.05").scaled_value))
         # TODO this is actually inv of solidity time stretch, fix
         expected_timestretch = _to_unscaled_decimal((1 / expected_timestretch_fp).scaled_value)
         expected_inv_timestretch = _to_unscaled_decimal(expected_timestretch_fp.scaled_value)
 
         expected_pool_config = {
             "contractAddress": hyperdrive_contract_addresses.mock_hyperdrive,
             "baseToken": hyperdrive_contract_addresses.base_token,
-            "initialSharePrice": _to_unscaled_decimal(int(1e18)),
-            "minimumShareReserves": _to_unscaled_decimal(int(10e18)),
+            "initialSharePrice": _to_unscaled_decimal(FixedPoint("1").scaled_value),
+            "minimumShareReserves": _to_unscaled_decimal(FixedPoint("10").scaled_value),
             "positionDuration": 604800,  # 1 week
             "checkpointDuration": 3600,  # 1 hour
             # TODO this is actually inv of solidity time stretch, fix
             "timeStretch": expected_timestretch,
             "governance": deploy_account.address,
             "feeCollector": deploy_account.address,
-            "curveFee": _to_unscaled_decimal(int(0.1e18)),  # 10%
-            "flatFee": _to_unscaled_decimal(int(0.0005e18)),  # 0.05%
-            "governanceFee": _to_unscaled_decimal(int(0.15e18)),  # 15%
+            "curveFee": _to_unscaled_decimal(FixedPoint("0.1").scaled_value),  # 10%
```

Comment:
_to_unscaled_decimal seems silly

Reply:
so you're called _to_unscaled_decimal on a scaled value, why not just covert straight to a string?  isn't there a string method on fp?

Reply:
yeah can't you just do `Decimal(str(FixedPoint("0.1")))`?

Reply:
I'm using the private function to do the above to keep the code a bit more readable

Reply:
Ah I see what you mean. Updating _to_unscaled_decimal to take in FixedPoint